README.md

```diff
@@ -1,2 +1,5 @@
 # Project_FE
-## This is my first project!
\ No newline at end of file
+## This is my first project!
+*** hello world ***
+ergeg
+gergergerg
```

Comment:
THis is redundant also. Remove it please

Reply:
zdelal